### ISSUE TITLE
Experimental results of given configs.

### ISSUE THREAD

**Issue by Abyssaledge** [benjaminrwilson/torchbox3d#1]
Nice work!
May I ask if it is convenient for you to provide the basic results of the three given configs (CenterPoint 50/100/200 ) for reference?
And since the point cloud range of 200x200 seems quite large, I am curious about the devices you use and the training memory footprint.

**Comment by benjaminrwilson** [comment]
Hi @Abyssaledge, thanks! The 200x200 meter configuration was used for the baseline here: https://eval.ai/challenge/1710/leaderboard/4078. One thing to keep in mind is that the evaluation config for the challenge only evaluates 150 meters (3D Euclidean distance) from the origin.

> And since the point cloud range of 200x200 seems quite large, I am curious about the devices you use and the training memory footprint.

Yes, it is quite large. I used a 8 32GB V100s for this model -- the memory footprint quickly can reach the full 32 GB depending on the batch size. I would recommend scaling down to 150 meters.

I'll try to train a few of the other configurations for reference performance.

**Comment by mi220314** [comment]
Also have some memory issue. May I ask if there is some updates on configs and memory for reference?

**Comment by benjaminrwilson** [comment]
Hi @mi220314, have you tried using one of the smaller grid size configs? Additionally, what batch size are you using?

**Comment by mi220314** [comment]
Yes, I have used a 50*50 grid size with batch size=4, it's fine now. Thanks for your advice~
